demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py

```diff
@@ -735,7 +737,11 @@ def poll_incident_state(
                 time.sleep(interval)
                 elapsed_time = int(time.time() - start_time)
 
-        raise RuntimeError(f"status of incident {incident_name} is {incident_status}")
+        raise PollTimeout(
```

Comment:
why not just raise the connection error? Think this will be more helpful for people trying to debug the real issue. this should be a log

Reply:
Connection Error isn't the case here, it just means the incident hasn't reached the required state.